test/integration/connectable/get-connection.js

```diff
@@ -24,6 +24,10 @@ describe('Connectable ::', function() {
       });
     });
 
+    after(function(done) {
+      manager.pool.end(done);
```

Comment:
Should we use the mp-postgresql `destroyManager` method for this?

Reply:
Up to you :slightly_smiling_face:

Reply:
Thinking about this more, I think the above approach is preferable to using the library under test for cleanup.